applications/doodle/src/doodle_exe.erl

```diff
@@ -253,7 +253,7 @@ init([Call]) ->
     process_flag('trap_exit', 'true'),
     CallId = whapps_call:call_id(Call),
     put('callid', CallId),
-    self() ! 'initialize',
+    gen_listener:cast(self(), 'initialize'),
```

Comment:
Yes the main difference is `spawn('cf_singular_call_hooks', 'maybe_hook_call', [CallWithHelpers]),` but I hoped this PR could help me understand the pattern/flow here